patterns/1-initial/public-administration-and-suppliers.md

```diff
@@ -0,0 +1,78 @@
+within the public administration and between this and its suppliers.
+The aim is to promote code sharing, reuse, and knowledge transfer to reduce costs, improve quality,
+and accelerate development.
+
+## Context
+
+Public administrations are typically slow making decisions, but when a decision is made, this stays
+for longer time. Decisions are conservative as they have to serve citizens, and they usually base most
+of the development effort in outsourced partners.
+
+The proposed context is the relationship between a public administration, all its suppliers, and the
+process to build a trusted collaboration environment. There are not official channels to accelerate
+certain developments while the backlog keeps growing over time, even when suppliers are willing to
+advance in some of those tasks.
+
+## Forces
+
+* Siloed Development: Teams often work in isolation, unaware of each other's work or potential
+synergies. This leads to redundant efforts, inconsistent code quality, and difficulties in maintaining
+and upgrading systems.
+* Lack of Standardization: There's a lack of standardized development practices, tools, and
+methodologies across teams. This results in inconsistent code quality, increased maintenance costs,
+and difficulties in onboarding new team members.
+* Limited Code Sharing: Code sharing is often restricted due to concerns about intellectual property,
+security, and organizational silos. This limits the potential for collaboration and knowledge sharing.
+* Bureaucratic Hurdles: Complex approval processes, bureaucratic red tape, and rigid organizational
+structures hinder innovation and slow down development cycles.
+* Insufficient Incentives: There's a lack of incentives to encourage developers to contribute to shared
+projects and collaborate with others.
+* Cultural Resistance to Change: Some individuals may resist changes to traditional development practices
+and may be hesitant to adopt new approaches like InnerSource.
+
+## Solutions
+
+To address these challenges, the organization may explore the adoption of InnerSource practices.
+
+Key elements of the proposed InnerSource approach include:
+
+* Shared Repositories: Establishing a central repository for code, documentation, and other artifacts.
+* Collaborative Development: Encouraging collaboration between teams and suppliers.
+* Transparent Processes: Implementing transparent processes for code review, testing, and deployment.
+* Incentives: Providing incentives to motivate developers to contribute to shared projects.
+* Clear Roles and Responsibilities: Defining roles and responsibilities for different stakeholders,
+such as project owners, developers, and reviewers.
+* Overcoming Legal and Organizational Hurdles: Addressing legal and organizational challenges, such
+as intellectual property rights and procurement regulations.
+
+## Resulting Context
+
+* Suppliers would start working in a more collaborative way as the legal framework has evolved and allow
```

Comment:
If I'm not mistaken there's two key pieces to this pattern that are specific to public vs. private players:
* Legal procurement challenges when doing InnerSource/Open Source in the public sector.
* Potentially highlighting economic incentives for InnerSource/ Open Source in the public sector.

Reply:
I wonder if we would need localized versions of this pattern as they may differ going from country to country.